index.js

```diff
@@ -0,0 +1,6 @@
+const NewsHeadlineView = require('./newsHeadlineView')
+const GuardianApi = require('./guardianApi')
+
+const api = new GuardianApi()
+const view = new NewsHeadlineView(api)
+
```

Comment:
you're not running the app (not sure if that's the right way to say that!) from here, is that on purpose?